src/controllers/accomodationController.js

```diff
@@ -76,7 +78,7 @@ const doUpdate = async (req, res) => {
       accommodation.name = req.body.name;
     }
     if (req.body.location) {
-      accommodation.locationId = req.body.location;
+      accommodation.locationId = changeToArray(req.body.location);
```

Comment:
locationId is an array? change to a better naming convention referencing a possible plural value